pkg/pidfile/pidfile.go

```diff
@@ -33,6 +36,7 @@ func New(path string) (*PIDFile, error) {
 	if err := checkPIDFileAlreadyExists(path); err != nil {
 		return nil, err
 	}
+	system.MkdirAll(filepath.Dir(path), os.FileMode(0644))
```

Comment:
No. If we want to use the default umask, we should use `0666`. If we want to make sure this is never world readable, we should be using `0660`.